magenta-lib/src/main/scala/magenta/deployment_type/AutoScaling.scala

```diff
@@ -127,18 +127,18 @@ object AutoScaling  extends DeploymentType {
   ){ (pkg, resources, target) =>
     implicit val keyRing = resources.assembleKeyring(target, pkg)
     implicit val artifactClient = resources.artifactClient
-    val reporter = resources.reporter
     val prefix = S3Upload.prefixGenerator(
-      stack = if (prefixStack(pkg, target, reporter)) Some(target.stack) else None,
-      stage = if (prefixStage(pkg, target, reporter)) Some(target.parameters.stage) else None,
-      packageName = if (prefixPackage(pkg, target, reporter)) Some(pkg.name) else None
+      stack = if (prefixStack(pkg, target, resources.reporter)) Some(target.stack) else None,
+      stage = if (prefixStage(pkg, target, resources.reporter)) Some(target.parameters.stage) else None,
+      packageName = if (prefixPackage(pkg, target, resources.reporter
+      )) Some(pkg.name) else None
     )
     List(
       S3Upload(
         target.region,
-        bucket(pkg, target, reporter),
+        bucket(pkg, target, resources.reporter),
         Seq(pkg.s3Package -> prefix),
-        publicReadAcl = publicReadAcl(pkg, target, reporter)
+        publicReadAcl = publicReadAcl(pkg, target, resources.reporter)
```

Comment:
There is no actual change here unless I missed something? I'd leave it as was if you're able.